card/mod/02_basic_types/set/all/base.rb

```diff
@@ -6,9 +6,20 @@ def show view, args
   end
 
   # NAME VIEWS
-
   simple_args = { closed: true, perms: :none }
-  view(:name,     simple_args) { card.name                           }
+  view :name, simple_args do |args|
+    return card.name unless args[:variant]
+    args[:variant].split(/[\s,]+/).inject(card.name) do |name, variant|
+      if ::Set.new([
+        :downcase, :upcase, :singularize, :pluralize, :capitalize,
+        :swapcase, :reverse, :succ
```

Comment:
ok, maybe I was to lazy with just using the method names. I'm going to deverbify them. I'm fine with the easter egg solution. There is also titleize in ActiveSupport. That's probably another option worth to mention in the documentation.